server/catalogue/routes/catalogueView.ts

```diff
@@ -21,7 +21,7 @@ export default class CatalogueView {
         text: this.presenter.text.searchByProgrammeNameInput.heading,
         classes: 'govuk-label--m',
       },
-      value: '', // this.presenter.fields.keywords,
+      value: this.presenter.filter.programmeName ? this.presenter.filter.programmeName : null,
```

Comment:
can we use coalescing 

```suggestion
      value: this.presenter.filter.programmeName ?? null,
```